### PR TITLE
Feature/marker module

### PR DESCRIPTION
**Pull request by ktalley-figure**

### CODE REVIEW

File: packages/bindings/src/types.rs [hunk 1]
```diff
@@ -141,10 +91,37 @@ pub struct AccessGrant {
 #[derive(Serialize, Deserialize, Clone, Debug, PartialEq, JsonSchema)]
 #[serde(rename_all = "snake_case")]
 pub enum MarkerPermission {
-    Burn,
-    Deposit,
-    Delete,
-    Grant,
-    Mint,
-    Withdraw,
+    AccessAdmin,
```
Comment (dpederson-figure) [inline packages/bindings/src/types.rs:94]: Although, these types should work for encoding messages, they will break on query - See https://github.com/provenance-io/provenance/blob/feature/add-marker-wasm/x/marker/wasm/types.go#L50

Comment (dpederson-figure) [reply packages/bindings/src/types.rs:94]: We should keep these as they were - ie without the `Access` prefix.

**Review by dpederson-figure — CHANGES_REQUESTED** [review]
Once the `Access` prefix is removed from the permission types, this should be good to go.

File: packages/mocks/testdata/marker.json [hunk 2]
```diff
@@ -1,13 +1,13 @@
 {
     "address": "tp18vmzryrvwaeykmdtu6cfrz5sau3dhc5c73ms0u",
     "coins": [],
-    "public_key": "",
     "account_number": 19,
     "sequence": 0,
     "manager": "tp15rrl3qjafxzlzguu5x29xh29pam35uetkpnnph",
     "accesscontrol": [],
```
Comment (dpederson-figure) [inline packages/mocks/testdata/marker.json:7]: Think this should be called "permissions" to match the rust type

Comment (ktalley-figure) [reply packages/mocks/testdata/marker.json:8]: fixed

File: packages/bindings/src/types.rs (hunk 1 above)
Comment (ktalley-figure) [reply packages/bindings/src/types.rs]: fixed

**Review by dpederson-figure — APPROVED** [review]
LGTM

**Pull request merged**